wiki/en/Running-a-Server.md

```diff
@@ -246,7 +250,8 @@ Set server not to record by default when recording is configured.
 
 Recording starts once the first person connects to the Server, and stops when the last person leaves.
 
-If the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new directory. SIGUSR2 will toggle recording on/off.
+If the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new directory. SIGUSR2 will toggle recording on/off. If [JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md) is enabled, you will also be able to manage the server in a more fine grained way comparable to the GUI. Please see the (experimental) [JSON-RPC documentation on the recorder](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md#jamulusserverstartrecording).
```

Comment:
```suggestion
If the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new directory. SIGUSR2 will toggle recording on/off. If [JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md) is enabled, you will also be able to manage the server in a way comparable to the GUI. Please see the (experimental) [JSON-RPC documentation on the recorder](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md#jamulusserverstartrecording).
```

Reply:
Hmm. In a way comparable to the GUI might be misleading as an API is not a GUI.

Reply:
The options controllable through the GUI are the same ones controllable through the API, so they should be _directly_ comparable.

Reply:
Fair point. We still have "in a way"